src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -376,85 +344,14 @@ public async Task DownloadQualityProfile_WhenBindingConfigIsNull_Fails()
         }
 
         [TestMethod]
-        public void GetBindingLanguages_ReturnsDistinctLanguagesForProjects()
+        public void GetBindingLanguages_ReturnsExpectedLanguagese()
```

Comment:
I simplified the existing test to check that the supported languages match the ones returned by the host.